include/crm/common/nvpair_internal.h

```diff
@@ -27,7 +27,11 @@ typedef struct unpack_data_s {
     const char *first_id;           // Block with this XML ID should sort first
     pcmk_rule_input_t rule_input;   // Data used to evaluate rules
 
-    // Whether each block's values should overwrite any existing ones
+    /* Whether each block's values should overwrite any existing ones
+     *
+     * @COMPAT Only external call paths set this to true; drop if we internalize
+     * pe_eval_nvpairs() and pe_unpack_nvpairs().
```

Comment:
They won't be made internal, but they will be deprecated and replaced with a single new public API that can ignore overwrite